### ISSUE TITLE
[Feature request] personalised "Recommended" section based on subscriptions & recommended

### ISSUE THREAD

**Issue by mdimura** [iv-org/invidious#4042]
**Problem**
After moving to invidious, I was missing the personalized video suggestions (youtube's start page). 

**Current solution (flawed)**
However, I found a workaround - the "play next" (recommended) section under the videos from my subscriptions. Of course, If I want to check the new suggestions, it's quite inconvenient to go over subscriptions just to check the "recommended" section.

**Suggested solution**
Could invidious automate this process? For example, it could iterate over my subscriptions, check first ~3 videos, and for each of those take top ~5 recommended videos, then merge everything up into a list and show this list next to "popular" and "trending"?

Thanks for maintaining the project, it's awesome!

**Comment by unixfox** [comment]
Invidious wants to stay as close as possible to "a vanilla" experience like you find on youtube. (Except with all of that tracking and bloatware not needed)

This means that we do not plan to implement a recommendation algorithm. The popular page is already very good for single user, it shows the more popular videos amongst your subscriptions.
I do note that on a public instance this doesn't work well per single user as it show all the popular videos amongst all the subscribed channels from all the users.

But you are free to "hack" the code on your side and run a custom invidious instance that do that, that's the beauty of open source.

**Comment by retrocaster** [comment]
@unixfox I'm running my own local private invidious instance and the "Popular" feed is basically identical to my "Subscriptions" feed (same videos in same order; I've only found a single video missing from "Popular")

Given that I see no point in "Popular" for someone running their own invidious instance.

When I used to use public instances I found "Popular" useful! Other invidious users have similar tastes so the recommendations were good. But I can't see that anymore with my current private instance.
